tests/ce/test_ce.py

```diff
@@ -0,0 +1,33 @@
+from openfisca_us_data import CE
+
+
+def test_ce_from_2019():
+    """Test the Consumer Expenditure data generating functions for 2019"""
+    ce = CE()
+    ce.generate(2019)
+    ce_2019 = ce.load(2019)
+
+    assert len(set(ce_2019.keys()).intersection({"annual", "household"})) == 2
+
+    # Test household average demographic estimate -------------------------
+    est_income_bf_tax = ce_2019["/annual/income_before_tax"][()]
+    assert round(est_income_bf_tax) == 82743
+
+    # Test household average expenditure estimate -------------------------
+    est_alcohol_expense = ce_2019["/annual/alcohol"][()]
+    assert round(est_alcohol_expense) == 536
+
+    # Test Household Sector CO2 emissions are "in the ballpark" -----
+    est_co2_kg_per_hh = ce_2019["/annual/co2_kg"][()]
+    hhs_in_us = 122.8e6
```

Comment:
```suggestion
    HHS_IN_US = 122.8e6
```
same with other constants

Reply:
Made them constants. Also decided to pull them out of the function. Let me know if you think they belong inside

Reply:
Changed my mind and left them in the function